CCDB/include/CCDB/CcdbApi.h

```diff
@@ -24,6 +24,12 @@
 #include <TMessage.h>
 #include "CCDB/CcdbObjectInfo.h"
 
+#if !defined(__CINT__) && !defined(__MAKECINT__) && !defined(__ROOTCLING__) && !defined(__CLING__)
```

Comment:
why do we need this?

Reply:
The libjalien isn't a ROOT library and doesn't have a ROOT dictionary. I noticed problems with ROOT interpreter when `CcdbApi.h` was included because it couldn't understand this type.

There is similar code in JAliEn-ROOT / AliEn-ROOT-Legacy, but maybe there is another way around it. In any case, if this header is not explicitly included in a ROOT interpreter session then maybe it works without it.

Reply:
@sawenzel What do you advise?

Reply:
No hard opinion here. But usually this should not be necessary. One can already use o2::ccdb::CcdbApi in a ROOT session without including the header. And the protection could be put outside as well.
We can also merge as is and remove later.